go_utils/photo_download.py

```diff
@@ -65,19 +78,28 @@ def download_all_photos(targets):
     targets : list of tuple of str
         Contains tuples that store the url, directory, and filename of the desired photos to be downloaded in that order.
     """
-    for target in targets:
-        download_photo(*target)
+    expectedNumParams = 3
+    if pd.isna(targets):
+        warnings.warn("Targets was none")
+    else:
+        for target in targets:
+            if (type(target) is tuple) and len(target) == expectedNumParams:
+                download_photo(*target)
+            else:
+                warnings.warn(f"Target incorrectly formatted: {target}")
 
 
 def _format_param_name(name):
+    if pd.isna(name):
+        return None
     return (
         "".join(s.capitalize() + " " for s in name.split("_"))
         .replace("Photo", "")
         .strip()
     )
 
 
-def get_mhm_download_targets(
+def get_mhm_download_targets(  # noqa: C901
```

Comment:
You could get around C901 by making "private functions" (see _format_param_name()) outside of this method. It might also read better if you do it that way.

Reply:
updated! -- Thanks for the tip!